src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -105,6 +111,14 @@ public SubscriptionService(final SubscriptionDbRepository subscriptionRepository
                 .map(Optional::get)
                 .forEach(eventType -> client.checkScopes(eventType.getReadScopes()));
 
+        if (featureToggleService.isFeatureEnabled(DISABLE_SUBSCRIPTION_CREATION)) {
+            try {
+                return Result.ok(getExistingSubscription(subscriptionBase));
```

Comment:
oh I see. This is ackward. If I wanted to implement the toggle in the service level I would have to "throw" an exception so that it would be caught in the controller and `processDuplicatedSubscription` would be called.

Problably I should implement this in the controller layer.